abdulrahman/style.css

```
@@ -0,0 +1,122 @@
+nav{
+    background-color: rgb(201, 228, 228);
+    width: 100%;
+    border-radius:6px;
+    position: fixed;
+    box-shadow: 7px 7px 20px #aaaaaa;
+    height:80px;
+    z-index: 9;
+    
+  }
+  .logo{
+    height:60px;
+    width:60px;
+    margin-top: 10px;
+    margin-left: 30px;
+  }
+  .blue{
+    text-decoration: none;
+    font-size: 24px;
+    border: 2px solid rgb(29,113,139);
+    border-radius: 10px;
+    padding: 8px 16px;
+    background-color: rgb(4,154,231);
+    color:white;
+    text-transform: uppercase;
+    font-family: 'Manrope', sans-serif;
+    margin: 5px;
+  }
+  .white{
+    text-decoration: none;
+    font-size: 24px;
+    border: 2px solid rgb(29,113,139);
+    border-radius: 10px;
+    padding: 8px 16px;
+    background-color: rgb(255, 255, 255);
+    color: rgb(29,113,139);
+    text-transform: uppercase;
+    font-family: 'Manrope', sans-serif;
+  }
+  .links{
+    float: right;
+    margin-top: 20px;
+    margin-right: 30px;
+  }
+  #cover{
+    position: relative;
+    background-size: cover;
+    width: 100%;
+    display: block;
+  }
+  .container{
+    width: 80%;
+          margin: 0 auto;
+          box-shadow: 7px 7px 20px #aaaaaa;
```

the indentation of the code is some thing really important so your code should be clean and readable 

not just here but every where in your code you should fix the indentation of the code 

```suggestion
  .container{
    width: 80%;
    margin: 0 auto;
    box-shadow: 7px 7px 20px #aaaaaa;
```